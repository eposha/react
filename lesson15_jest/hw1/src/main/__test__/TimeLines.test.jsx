import React from "react";
import { shallow } from "enzyme";
import TimeLines from "../TimeLines";

describe("<TimeLines/ >", () => {
  it("should display TimeLines", () => {
    const wrappedComponent = shallow(<TimeLines />);
    expect(wrappedComponent).toMatchSnapshot();
  });
});
