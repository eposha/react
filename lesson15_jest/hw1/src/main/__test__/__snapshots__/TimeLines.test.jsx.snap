// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TimeLines/ > should display TimeLines 1`] = `
<div
  className="time__lines"
>
  <div
    className="time__line"
    key="0"
  />
  <div
    className="time__line"
    key="1"
  />
  <div
    className="time__line"
    key="2"
  />
  <div
    className="time__line"
    key="3"
  />
  <div
    className="time__line"
    key="4"
  />
  <div
    className="time__line"
    key="5"
  />
  <div
    className="time__line"
    key="6"
  />
  <div
    className="time__line"
    key="7"
  />
  <div
    className="time__line"
    key="8"
  />
  <div
    className="time__line"
    key="9"
  />
  <div
    className="time__line"
    key="10"
  />
  <div
    className="time__line"
    key="11"
  />
  <div
    className="time__line"
    key="12"
  />
  <div
    className="time__line"
    key="13"
  />
  <div
    className="time__line"
    key="14"
  />
  <div
    className="time__line"
    key="15"
  />
  <div
    className="time__line"
    key="16"
  />
  <div
    className="time__line"
    key="17"
  />
  <div
    className="time__line"
    key="18"
  />
  <div
    className="time__line"
    key="19"
  />
  <div
    className="time__line"
    key="20"
  />
  <div
    className="time__line"
    key="21"
  />
  <div
    className="time__line"
    key="22"
  />
  <div
    className="time__line"
    key="23"
  />
</div>
`;
